packages/core/src/types.ts

```diff
@@ -138,6 +138,7 @@ export interface UseFormReturn<Values extends FormValues> {
   setFieldValue: UseFormSetFieldValue<Values>;
   setErrors: (errors: FormErrors<Values>) => void;
   setFieldError: UseFormSetFieldError<Values>;
+  setFieldTouched: (name: Values, touched?: boolean) => void;
```

Comment:
The type definition here seems incorrect. See `UseFormSetFieldError<Values>` for details on this section.

https://github.com/Mini-ghost/vorms/blob/009ff62c84a6959343505f3e4ea9b63d71f3f2ee/packages/core/src/types.ts#L110-L115

Reply:
corrected but I don't know if it is necessary to return errors :)

Reply:
Thank you for your contribution. Regarding returning errors, do you think we should be consistent with `setFieldValue`, `setFieldError`?

Reply:
On the one hand, it's easier, on the other hand we don't need the extra, so I suggest we keep it that way for now (i.e. forever).

Reply:
I'm not sure I understand what you mean. Do you think something like below is better? 

```ts
export type UseFormSetFieldTouched<Values extends FormValues> = <
  Name extends Path<Values>,
>(
  name: Name,
  touched: boolean,
) => void
```

Because it will be consistent with other functions.

Reply:
yes maybe better, could have corrected it as you like ?)

Reply:
Thank you so much for your assistance in clarifying the issue. Considering the situation, I believe it might be more appropriate to return `void` for the time being.